src/segment_media_windows.go

```diff
@@ -0,0 +1,30 @@
+// +build windows
+
+package main
+
+import (
+	"encoding/json"
+	"os/exec"
+)
+
+func (s *media) enabled() bool {
+	var cmd = exec.Command("sys-media-info")
+	var str, err = cmd.Output()
+	if err == nil {
+		json.Unmarshal(str, &s.info)
+		return true
+	}
+	s.other, err = s.env.getWindowTitle("cloudmusic.exe", "^(.*\\s-\\s.*)$")
```

Comment:
This can be refactored to a for loop using the players and be easily extended that way too. Right now this is basically 3 times the same code.